packages/lib-classifier/src/store/TranscriptionReductions/TranscriptionReductions.spec.js

```diff
@@ -0,0 +1,208 @@
+        expect(annotation.frame).to.equal(2)
+      })
+      reductionsModel.changeFrame(0)
+    })
+
+    describe('transcribed lines', function () {
+      let transcribedLine
+
+      before(async function () {
+        transcribedLine = reductionsModel.transcribedLines[0]
+      })
+
+      it('should have text options', function () {
+        expect(transcribedLine.textOptions).to.deep.equal([
+          'Here are some test',
+          'Here are some test',
+          'Here are some test'
+        ])
+      })
+
+      it('should have two points', function () {
+        expect(transcribedLine.points).to.have.lengthOf(2)
+      })
+
+      it('should have a start point', function () {
+        const x = 32.550689697265625
+        const y = 297.0990905761719
+        expect(transcribedLine.points[0]).to.deep.equal({ x, y })
+      })
+
+      it('should have an end point', function () {
+        const x = 989.6483154296875
+        const y = 280.3498840332031
+        expect(transcribedLine.points[1]).to.deep.equal({ x, y })
+      })
+    })
+  })
+
+  describe('without transcribed lines', function () {
+    const caesarClient = new GraphQLClient('https://caesar-staging.zooniverse.org/graphql')
+    let reductionsModel
+
+    before(async function () {
+      const response = {
+        workflow: {
+          subject_reductions: [{ data: reducedEmptySubject }]
+        }
+      }
+      sinon.stub(caesarClient, 'request').callsFake(() => Promise.resolve(response))
+      reductionsModel = TranscriptionReductions.create({
+        caesarReducerKey: 'ext',
+        subjectId: '13971170',
+        workflowId: '5339'
+      }, {
+        client: {
+          caesar: caesarClient
+        }
+      })
+      await reductionsModel.fetchCaesarReductions()
+    })
+
+    after(function () {
+      caesarClient.request.restore()
+    })
+
+    it('should exist', function () {
+      expect(reductionsModel).to.be.ok()
+    })
+
+    it('should not have any transcribed lines', function () {
+      reductionsModel.reductions.forEach(reduction => expect(reduction.data.transcribed_lines).to.equal(0))
+      expect(reductionsModel.transcribedLines).to.be.empty()
+    })
+  })
+
+  describe('without a configured reducer', function () {
+    const caesarClient = new GraphQLClient('https://caesar-staging.zooniverse.org/graphql')
+    let reductionsModel
+
+    before(async function () {
+      sinon.stub(console, 'error')
+      const error = new Error('Error: GraphQL Error (Code: 404)')
+      error.response = {
+        error: '',
+        status: 404
+      }
+      error.request = {
+        query: '{ workflow(id: 3389) { subject_reductions(subjectId: 13971170, reducerKey:"ext") { data } } }',
+        variables: undefined
+      }
+      sinon.stub(caesarClient, 'request').callsFake(() => Promise.reject(error))
+      reductionsModel = TranscriptionReductions.create({
+        caesarReducerKey: 'ext',
```

Comment:
```suggestion
        caesarReducerKey: 'alice',
```

Reply:
I think this can be deleted.

Reply:
I've deleted that line. The reducer key is a constant for all transcription subjects, so shouldn't need to be passed in the initial snapshot.